accessibility-checker-engine/help/landmark_name_unique.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "landmark_name_unique: Accessibility Checker Rule Help"
+---
+import "../../../styles/ToolHelp.scss"
+import { CodeSnippet, Tag } from "carbon-components-react";
+
+<div className="toolHelp">
+<Row>
+<Column colLg={16} colMd={8} colSm={4} className="toolHead">
+
+### Multiple landmark roles with the same `role` attribute are not disambiguated with a unique label or different parent  
+
+<div id="locLevel"></div>
+
+Multiple landmarks with the same `role` should have a unique `aria-labeledby` or `aria-label` or be nested in different parents
+
+</Column>
+</Row>
+<Row>
+<Column colLg={11} colMd={5} colSm={4} className="toolMain">
+
+### Why is this important?
+
+Regions (sections) on a page may contain headings, content, white space, and even nested regions for users to better consume the content. 
+Landmarks for the regions help assistive technology (AT) users orient themselves to the page and help them easily navigate to various regions of a page. 
+When multiple regions with the same role exist on the same page, people using screen readers, voice control, 
+and other ATs will have difficulty identifying or disambiguating between them 
```

Comment:
```suggestion
and other ATs will have difficulty identifying or disambiguating between them unless they have unique labels
```

Reply:
and the "navigation" landmark was reported as the more common occurence.